jobs/build_docker/build_docker.groovy

```diff
@@ -3,6 +3,13 @@ node(build_docker_node){
         withEnv([
             "MANIFEST_FILE_URL=${env.MANIFEST_FILE_URL}",
             "IS_OFFICIAL_RELEASE=${env.IS_OFFICIAL_RELEASE}",
+            "BINTRAY_SUBJECT=${env.BINTRAY_SUBJECT}",
```

Comment:
why docker build requires those .... I think docker are build from src code (except the on-imagebiulder ?)